CogniteSdk.Types/Common/CursorQueryBase.cs

```diff
@@ -28,7 +28,7 @@ public override string ToString()
         }
 
         /// <inheritdoc/>
-        public List<(string, string)> ToQueryParams()
+        public virtual List<(string, string)> ToQueryParams()
```

Comment:
I think that `ToQueryParams` should probably be marked with `override` in the other subclasses as well. E.g ThreeDModelQuery, ThreeDNodeQuery, ThreeDRevisionQuery and RawRowQuery. I know it's now part of this PR, but if you agree we can fix this in this PR.